src/Dialog/Content/Panel.tsx

```diff
@@ -97,8 +98,8 @@ const Panel = React.forwardRef<ContentRef, PanelProps>((props, ref) => {
   let closer: React.ReactNode;
   if (closable) {
     closer = (
-      <button type="button" onClick={onClose} aria-label="Close" className={`${prefixCls}-close`}>
-        {closeIcon || <span className={`${prefixCls}-close-x`} />}
+      <button type="button" onClick={onClose} title={closeLabel} aria-label={closeLabel ?? "Close"} className={`${prefixCls}-close`}>
+        <span className={`${prefixCls}-close-x`} aria-hidden={closeLabel !== undefined ? true : undefined}>{closeIcon}</span>
```

Comment:
The change to `closeIcon` looks like a break change.

Reply:
because of rendering of the span? Well, from the aria point of view, that change would be correct. The button has already all info for aria, the closeIcon is for visibility only that is why thre is aria-hidden.

It is also possible to keep the line 102 as it was. Then using antd overwrite of closeIcon is a must, to set aria-hidden as long as antd doesn't set aria-hidden on the span automatically. 
It would be already an improvement. Should I change the line in my PR?